biz.aQute.bnd.gradle/src/aQute/bnd/gradle/Export.groovy

```diff
@@ -226,6 +236,7 @@ public class Export extends DefaultTask {
         }
       } finally {
         logReport(run, logger)
+        run.setParent(parent)
```

Comment:
Not sure I see the utility in restoring the parent just before the run is closed. I would just avoid this.